### ISSUE TITLE
Make docker work again with ZeppelinOS

### ISSUE THREAD

**Issue by H34D** [oceanprotocol/keeper-contracts#266]
https://github.com/oceanprotocol/keeper-contracts/blob/841d4135a117614990a15e51e4d5299120152860/scripts/keeper.sh#L13

does not work anymore.

**Comment by billbsing** [comment]
I've found it too, cannot get any contracts copied over when running travis

```
keeper-contracts_1_92dcc2962c11 | 
keeper-contracts_1_92dcc2962c11 | > @oceanprotocol/keeper-contracts@0.5.3 clean /keeper-contracts
keeper-contracts_1_92dcc2962c11 | > rm -rf ./build/ ./artifacts/*.development.json ./coverage/ ./coverageEnv/ ./coverage.json
keeper-contracts_1_92dcc2962c11 | 
keeper-contracts_1_92dcc2962c11 | 
keeper-contracts_1_92dcc2962c11 | > @oceanprotocol/keeper-contracts@0.5.3 migrate /keeper-contracts
keeper-contracts_1_92dcc2962c11 | > echo 'migrations are deprecated! use zos instead' "--network=spree"
keeper-contracts_1_92dcc2962c11 | 
keeper-contracts_1_92dcc2962c11 | migrations are deprecated! use zos instead --network=spree
```

**Comment by H34D** [comment]
closes with #220
